datamodel/src/main/java/com/hartwig/serve/datamodel/ClinicalTrial.java

```diff
@@ -32,5 +32,14 @@ public abstract class ClinicalTrial implements Intervention {
     public abstract Set<String> therapyNames();
 
     @Nullable
-    public abstract String genderCriterium();
-}
\ No newline at end of file
+    public abstract GenderCriterium genderCriterium();
+
+    @NotNull
+    public abstract Set<Indication> indications();
+
+    @NotNull
+    public abstract Set<MolecularCriterium> molecularCriteria();
```

Comment:
Dumb question, the semantics are these must all be satisfied, or any? I recall seeing some ticket mentioning both complex and combined profiles as different things and wasn't sure how they needed to be modelled.

Reply:
For a clinical trial you should read this set as 'OR', since for example for a trial you should have an EGFR amp OR an EGFR fusion). So not all `Set<MolecularCriterium>` have to be satisfied.

If we have combined evidence (e.g. “ALK fusion” +  “ALK G1202R”) this will be in 1 `MolecularCriterium` with in `fusions()` one `ActionableFusion` and in `hotspots()` one `ActionableHotspot`. So _within_ a `MolecularCriterium` you should read it as 'AND'.

Didn't know about the ticket, thanks for pointing me to it. 
Complex evidence as explained in this ticket, is not modelled by this new datamodel. Me talking about "complex profiles" before was probably confusing, let's talk about "combined evidence" from now on when we talk about e.g. “ALK fusion” +  “ALK G1202R” :)

Reply:
In SERVE datamodel we use "combined evidence" while CKB uses "complex profiles" for the same thing, so that may cause the confusion :) 

Wondering if we could clarify the fact that the criteria should be or'ed in the name, don't have inspiration though (`anyMolecularCriteria`?)